cmd/subnetcmd/configure.go

```diff
@@ -36,33 +45,58 @@ func configure(cmd *cobra.Command, args []string) error {
 	subnetName := chains[0]
 
 	const (
-		subnetConf = "Subnet config"
-		chainConf  = "Chain config"
+		chainLabel  = constants.ChainConfigFileName
+		subnetLabel = constants.SubnetConfigFileName
 	)
+	configsToLoad := map[string]string{}
 
-	options := []string{subnetConf, chainConf}
-	selected, err := app.Prompt.CaptureList("Which configuration file would you like to update?", options)
-	if err != nil {
-		return err
+	if subnetConf != "" {
+		configsToLoad[subnetLabel] = subnetConf
 	}
-	switch selected {
-	case subnetConf:
-		err = updateConf(subnetName, constants.SubnetConfigFileName)
-	case chainConf:
-		err = updateConf(subnetName, constants.ChainConfigFileName)
+	if chainConf != "" {
+		configsToLoad[chainLabel] = chainConf
 	}
-	if err != nil {
-		return err
+
+	// no flags provided
+	if len(configsToLoad) == 0 {
+		options := []string{chainLabel, subnetLabel}
+		selected, err := app.Prompt.CaptureList("Which configuration file would you like to provide?", options)
+		if err != nil {
+			return err
+		}
+		configsToLoad[selected], err = app.Prompt.CaptureExistingFilepath("Enter the path to your configuration file")
+		if err != nil {
+			return err
+		}
+		var other string
+		if selected == chainLabel {
+			other = subnetLabel
+		} else {
+			other = chainLabel
+		}
+		yes, err := app.Prompt.CaptureNoYes(fmt.Sprintf("Would you like to provide the %s file as well?", other))
+		if err != nil {
+			return err
+		}
+		if yes {
+			configsToLoad[other], err = app.Prompt.CaptureExistingFilepath("Enter the path to your configuration file")
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	// load each provided file
+	for f, c := range configsToLoad {
```

Comment:
could you use `file` and `config` instead of f and c, I was a little confused at first

Reply:
Used `fileName` and `configPath`